server/index.js

```diff
@@ -10,16 +10,23 @@ app.use(express.urlencoded({ extended: true }));
 
 app.get('/locations/:productID', (req, res) => {
   let productID = req.params.productID;
-  console.log('😛', productID)
   if(productID > 100) {
     res.sendStatus(404)
   }
   db.getOneProdLoc(productID)
     .then(data => res.json(data))
-    // .then(data => console.log('🤗 on the server index.js', data))
-    // .then(data => res.send(data))
     .catch(error => res.sendStatus(404))
-    //if error send status code of  404
+})
+
+app.get('/materials/:productID', (req, res) => {
+  let productID = req.params.productID;
+  console.log('😛 on the materials', productID)
+  if(productID > 100) {
```

Comment:
is this necessary? will your error handling in the catch cover this edge case if product 101 isn't found in the db?

Reply:
I guess it really is not needed, on the db I am already doing error handling.